sdk/eventhub/azure-eventhub-checkpointstoreblob-aio/azure/eventhub/extensions/checkpointstoreblobaio/_blobstoragecsaio.py

```diff
@@ -59,6 +61,11 @@ def from_connection_string(
             access key values. The value can be a SAS token string, an account shared access
             key, or an instance of a TokenCredentials class from azure.identity.
             Credentials provided here will take precedence over those in the connection string.
+        :keyword str api_version:
```

Comment:
Still documenting the `api_version` parameter here

Reply:
This isn't a problem any more since "api_version" is added back